libcontainer/mount_linux.go

```diff
@@ -1,7 +1,9 @@
 package libcontainer
 
 import (
+	"io/fs"
 	"strconv"
+	"syscall"
```

Comment:
(Nit: I prefer using unix for everything but it doesn't really matter.)

Reply:
Same -- I think with Go2 there will be no syscall and we'll have to switch to x/sys/unix anyway, so better do it now.